server/ansible/roles/pbench-server-crontab-disable/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: "save the current crontab before disabling"
+  shell:
+    cmd: "crontab -l > {{ pbench_server_install_dir }}/lib/crontab/crontab.back.{{ ansible_date_time.iso8601 }}"
+  when: ansible_user_id == pbench_user
+
+- name: "disable crontab"
+  command:
+    cmd: crontab -r
+  when: ansible_user_id == pbench_user
```

Comment:
Q: what happens here when `ansible_user_id` != `pbench_user`?? Does the whole role just silently do nothing?

I suppose the Ansible task output will say that it was skipped, but I wonder if anyone will actually notice... ;-)